server/routes/api/messages.js

```diff
@@ -36,11 +36,27 @@ router.post("/", async (req, res, next) => {
       senderId,
       text,
       conversationId: conversation.id,
+      read,
     });
     res.json({ message, sender });
   } catch (error) {
     next(error);
   }
 });
 
+router.put("/:messageId", async (req, res, next) => {
+  const { messageId } = req.params;
+  const { read } = req.body
+  //console.log(`update request message id ${messageId}`);
+  try{
+    const message = await Message.findByPk(messageId)
+    message.read = read;
+    message.save();
```

Comment:
Minor nitpick: the same thing can be achieved with single statement `Message.update()`. I think this code is clear enough, but wanted to let you know there is an alternative.

Reply:
Got it. Keeping it as it is for now.